barcode_validation/readme/USAGE.md

```diff
@@ -0,0 +1,19 @@
+This module is an abstract module. You can configure barcode validation, but to enable this feature, you need to install an extra module for a given model. This repository provide 'product_barcode_validation' and 'stock_barcode_validation' to validate barcodes for products, stock lots, product packagings, ...
```

Comment:
thought: is there really any use of barcode validation in a lower stack than the `product` module? Couldn't you just merge `barcode_validation` into `product_barcode_validation` and consider the latter one the base for others?